### ISSUE TITLE
v4.6.0 Not Working

### ISSUE THREAD

**Issue by UltiRequiem** [carloscuesta/gitmoji-cli#672]
Hello @carloscuesta!

Issue:

I just updated gitmoji, no command works.
Neither `-c` or` --commit`, or the others.

**OS**: Archlinux
**gitmoji -v**: v4.6.0
**Node version**: v16.8.0

**Error Message**:
```
/home/suzaku/.config/yarn/global/node_modules/gitmoji-cli/lib/utils/getEmojis.js:12
var _ora = _interopRequireDefault(require("ora"));
                                  ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /home/suzaku/.config/yarn/global/node_modules/gitmoji-cli/node_modules/ora/index.js from /home/suzaku/.config/yarn/global/node_modules/gitmoji-cli/lib/utils/getEmojis.js not supported.
Instead change the require of index.js in /home/suzaku/.config/yarn/global/node_modules/gitmoji-cli/lib/utils/getEmojis.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/home/suzaku/.config/yarn/global/node_modules/gitmoji-cli/lib/utils/getEmojis.js:12:35)
    at Object.<anonymous> (/home/suzaku/.config/yarn/global/node_modules/gitmoji-cli/lib/commands/commit/index.js:10:41)
    at Object.<anonymous> (/home/suzaku/.config/yarn/global/node_modules/gitmoji-cli/lib/commands/index.js:8:38)
    at Object.<anonymous> (/home/suzaku/.config/yarn/global/node_modules/gitmoji-cli/lib/cli.js:15:40) {
  code: 'ERR_REQUIRE_ESM'
}
```

![image](https://user-images.githubusercontent.com/71897736/131882123-18d5a5e9-4579-418f-8f4d-98f623db7216.png)

**Comment by carloscuesta** [comment]
Thanks for the report should be fixed in v4.6.1

**Comment by UltiRequiem** [comment]
Thanks!
